Tests/PSModuleGallery.Type.Tests.ps1

```diff
@@ -886,4 +888,85 @@ InModuleScope 'PSDepend' {
+
+    Describe "DotnetSdk Type PS$PSVersion" {
+        $IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
+        $GlobalDotnetSdkLocation = if ($IsWindowsEnv) { "$env:LocalAppData\Microsoft\dotnet" } else { "$env:HOME/.dotnet" }
+        $SavePath = '.dotnet'
+
+        Context 'Installs Dependency' {
+            $Dependency = Get-Dependency @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1"
+            It 'Parses the DotnetSdk dependency type' {
+                $Dependency | Should -Not -BeNullOrEmpty
+                $Dependency.DependencyType | Should -Be 'DotnetSdk'
+                $Dependency.Version | Should -Be '2.1.300'
+                $Dependency.DependencyName | Should -Be 'release'
+                $Dependency.Target | Should -Be $SavePath
+            }
+
+            It 'Installs the .NET Core SDK to the specified directory' {
+                Mock Test-Dotnet { return $false }
+
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1" -Force
+                Test-Path $SavePath | Should -BeTrue
+            }
+
+            It 'Does nothing if the .NET Core SDK is found' {
+                Mock Test-Dotnet { return $true }
+                Mock Install-Dotnet 
+
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1" -Force
+                Assert-MockCalled -CommandName Install-Dotnet -Times 0 -Exactly
+            }
+
+            AfterAll {
+                Remove-Item -Force -Recurse $SavePath -ErrorAction SilentlyContinue
+            }
+        }
+
+        Context 'Tests Dependency' {
+            # used to see if 'dotnet' is already on the PATH - we need this to return false
+            Mock Get-Command { return $false } -ParameterFilter { $Name -eq 'dotnet' }
+
+            It 'Can propertly compare semantic versions' {
+                Mock Test-Path { return $true } -ParameterFilter  { $Path -eq $GlobalDotnetSdkLocation }
+                Mock Get-DotnetVersion { return '2.1.330-rc1' }
+
+                # '2.1.330-rc1' >= '2.1.330-preview1'
+                # '2.1.330-rc1' >= '2.1.330-rc1'
+                # '2.1.330-rc1' >= '1.0'
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.semanticversion.depend.psd1" -Test -Quiet | Should -BeTrue
+            }
+        }
+
+        Context 'Imports Dependency' {
+            # used to see if 'dotnet' is already on the PATH - we need this to return false
+            Mock Get-Command { return $false } -ParameterFilter { $Name -eq 'dotnet' }
+
+            BeforeAll {
+                $originalPath = "$env:PATH"
+            }
+
+            It 'Can add the Target of the .NET Core SDK to the PATH' {
+                Mock Test-Dotnet { return $true }
+                Invoke-PSDepend @Verbose -Path "$TestDepends\dotnetsdk.complex.depend.psd1" -Force -Import -ErrorAction Stop
+                
+                $env:PATH | Should -Match "$SavePath*"
```

Comment:
`Should -Match` using regular expressions.  Should `"$SavePath*"` be `"$SavePath.*"`?